spec/factories/fact_checkpoints.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :fact_checkpoint do
+    simulation JSON.generate sample: {values: [1], errors: []}, stats: {mean: 1, stdev: 0, length: 1}
```

Comment:
Can we validate that the stats have percentiles?  Maybe 5th and 95th or adjusted 5th and 95th, maybe with 50th as well?  This seems more useful than values for some of this.